tests/test_roads.py

```diff
@@ -7,32 +7,25 @@
 
 import os
 
+import arcpy
 import pytest
 from arcgisscripting import ExecuteError  # pylint: disable=no-name-in-module
 
-import arcpy
-
 table_name = 'Roads_Edit'
-sde = os.path.join(os.path.dirname(__file__), '..', 'pro-project', 'localhost.sde')
+sde = os.path.join(os.path.dirname(__file__), '..', 'pro-project', 'localhost@utrans.sde')
 TABLE = os.path.join(sde, table_name)
```

Comment:
**nitpick**: we should switch this to use the newer [pathlib](https://docs.python.org/3/library/pathlib.html) which is the recommended library for the future.

something like the following should suffice
```suggestion
from pathlib import Path

sde = Path(__file__).parent / '..' / 'pro-project', 'localhost@utrans.sde'
TABLE = str(sde / table_name)
```